gradle/versions.gradle

```diff
@@ -37,11 +37,11 @@ dependencyManagement {
     dependency 'io.libp2p:jvm-libp2p:1.2.2-RELEASE'
     dependency 'tech.pegasys:jblst:0.3.15'
     dependency 'io.consensys.protocols:jc-kzg-4844:2.1.1'
-    dependency 'io.github.crate-crypto:java-eth-kzg:0.7.1'
+    dependency 'io.github.crate-crypto:java-eth-kzg:0.8.0'
```

Comment:
@zilm13 do we need any specific benchmark testing before updating this dependency?